### ISSUE TITLE
UCP webpage contributions and improvements

### ISSUE THREAD

**Issue by patel-nikhil** [Sh0wdown/UnofficialCrusaderPatch#574]
The newly added webpage for the UCP (per #555) is currently present but still a work in progress.
So far the following have been observed to be discussed and/or needs improvement:
- Update installation instructions to reflect new patcher file structure
 - for the "Silent installation steps", what is the behaviour when just executing the cli.exe directly
- Review auto-scrolling of page when reviewing installation steps
  - the page scrolls (down) so that the clicked button is at the very top of the screen
  - now if I've clicked step 3 button and want to go back to step 2, I have to manually scroll up a bit before I can see step 2 again
- Fix typo in aic/aiv installation regarding backup folders
- Note behaviour for aiv installation and backup
- for the FAQ page, video title text is in the same line as the video, which looks a bit weird... maybe add a line break in between there?
  - when viewed on 17" screen
- @Krarilotus : maybe we could also plug a german tutorial by Hacksülze in there, or the one where i designed the Igel castle
- @Heroesflorian: for "Features", all selectable entries from the sub-menu are on the same page, but for "AI Character Introduction" and "AI Castle Introduction"; each entry from the sub-menu has its own page. one may discuss which behaviour is preferable, but I'd argue it should be consistent

Re: features:
- @patel-nikhil let's maybe
- remove the entries for features from navbar
- make the cards for feature a bit smaller, to reduce wasted space and to see more at a single time
- try and figure out how to show/hide based on category (ie. filter?)

**Comment by Heroesflorian** [comment]
On discord questions around UCP multiplayer compatibility appear every now and then, so a dedicated section with a couple sentences regarding what prerequisites and limitations exist regarding UCP with multiplayer would be useful to have I think. 
(For example, changing player colors is something that can lead to crashes, and using different UCP settings or spectator mode may also lead to crashes or desync.)

**Comment by GRhin** [comment]
I think that is part of the FAQ? At least it was included as part of my
suggested questions to include, however player color isnt mentioned in my
answer.

**Comment by Krarilotus** [comment]
I have a suggestion: As we currently only fully support v1.41 of the Crusader game, i would suggest to link to the free HD patch from FireflyStudios to be installed for people who have still the v1.1 or v1.2 version of the game. Link:
http://www.strongholdcrusaderhd.com/game.html

also for those who didn't read #555  here is a link to the page that gets discussed:
https://patel-nikhil.github.io/UnofficialCrusaderPatch/index.html

where you can access the branch here:
https://github.com/Sh0wdown/UnofficialCrusaderPatch/tree/gh-pages

**Comment by patel-nikhil** [comment]
Yes, the mention about multiplayer was in the FAQ. I added a couple lines on the homepage with PR #577 because of how common the question is.
I've also added link to 1.41 HD patch per Krarilotus' suggestion.
And made other various improvements throughout.
